src/main/java/heronarts/lx/audio/FourierTransform.java

```diff
@@ -137,14 +137,38 @@ public FourierTransform compute(float[] samples) {
       this.real[i] = samples[bri] * this.windowCoefficients[bri];
       this.imaginary[i] = 0f;
     }
+
+    computeComplexNum();
```

Comment:
This doesn't really seem like an improvement to me... the function isn't called anywhere else, and for someone to follow the whole algorithm they now read code above this, jump down to read the function below, then come back up to read the code after the function call. If it was called elsewhere I think that'd be sensible to decompose, but as it's not, I think this is better inline.